src/content/core/namespaces.mdx

```diff
@@ -59,35 +64,45 @@ To share a namespace go to the Okteto dashboard, select the namespace you want t
 
 In the `Share` dialog, type the email address of the team members you want to share this namespace with. Once you press save, Okteto will update the necessary permissions and notify your teammates via email.
 
-## Transfer namespace ownership
 
-Okteto administrators have the ability to transfer non-personal namespaces between users. This process can be achieved through the Okteto UI.
+## Namespace administration
 
-To transfer a namespace to a new owner, go to the Okteto UI -> Admin -> Namespaces then locate the namespace that you would like to transfer. Click on the three dots `...` to display the namespace menu. From the namespace menu select `Transfer Ownership`.
+The Okteto platform will take care of all the required RBAC roles and bindings, resources [quotas](self-hosted/helm-configuration.mdx#quotas) and limits, and [network policies](self-hosted/helm-configuration.mdx#networkpolicies) so everyone in your team can safely deploy and develop applications in Kubernetes.
+
+
+### Transfer Namespace ownership
+
+Okteto administrators have the ability to transfer non-personal Namespaces between users. This process can be achieved through the Okteto UI and can be helpful for when you need to manually remove resources from an inactive user.
```

Comment:
Do you think we should emphasize that personal namespaces can't be transferred? Right now we mention in passing but we're not overly clear